grails-app/conf/Config.groovy

```diff
@@ -381,6 +381,120 @@ openboxes.dashboard.catalogsSummary.enabled = true
 // Default value for news summary
 openboxes.dashboard.newsSummary.newsItems = []
 
+openboxes {
+    tablero {
+        enabled = true
+        endpoints {
+            number {
+                inProgressPutaways {
+                    endpoint = "getInProgressPutaways"
```

Comment:
This is fine for now, but eventually I want this to become a URL so that it can be configured to point elsewhere. I don't have a specific use case for that now, but I could imagine a few scenarios that would allow us to use an API gateway to deal with testing, performance, monitoring, security, and maybe even point to a proper analytics data store in the future.

Reply:
I also want to have the number endpoint become separate endpoints for each component. In fact, every widget should be getting their own data.

Reply:
I can modify the endpoints to have the full URL, and I separated the number endpoints in this PR